volontulo/templates/emails/offer_application.html

```diff
@@ -0,0 +1,5 @@
+Email wolontariusza: {{ email }}<br />
+Numer telefonu: {{ phone_no }}<br />
+Imię i nazwisko: {{ fullname }}<br />
+Uwagi: {{ comments }}<br />
+ID oferty: <a href="{{ offer_url }}">{{ offer_id }}</a><br />
```

Comment:
Change anchor text to something more meaningful, ex. "Offer ID URL" - I did it wrong just putting only ID.
EOL